site_scons/options.py

```diff
@@ -90,6 +90,11 @@
         default=False,
         help="Setting this flag builds fasttree pngs for small-med sized trees")
 
+Script.AddOption('--write-pruned-partis-outfile',
```

Comment:
Seems like this is mostly (always?) being done in order to run linearham. Is there maybe a better way of making this clear?

Reply:
--write-linearham-yaml-input ?

Reply:
Sure; That sounds good. If you think there's value in just writing out the pruned partis file independent of wanting to run linearham, you could have two separate flags. Your call.

Reply:
If we need that later it shouldn't be hard to add separate flag then, but I'll wait until there is some use case.